recipe/meta.yaml

```
@@ -40,7 +49,6 @@ outputs:
         - {{ compiler("cxx") }}
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
         - {{ stdlib("c") }}
-        - patchelf <0.18.0                      # [linux]
       host:
```

Currently we don't have `arm-variant` in `host`, which I think we will want to add

There are other `outputs` that will need similar treatment

```suggestion
      host:
        - arm-variant * {{ arm_variant_type }}  # [aarch64]
```